src/node_modules/react/lib/getNextDebugID.js

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
(That's the same reason we don't allow absolute imports via Webpack config in CRA 😄 )